style.css

```diff
@@ -11,10 +11,12 @@
     src: url(/fonts/Poppins-Bold.ttf)
 }   
 :root {
-    --main-bg-color: #ffffff;   
+    --main-bg-color: #ffffff;
+    --footer-bg-color: #dedeebe0;   
+    --element-color: #f3efe6;
```

Comment:
What element?

Reply:
I used this light gray color for different elements. Don't know how name it. 
Can change to --light-grey-color: #f3efe6;